src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+class TonMacronFriendInvitationAdmin extends AbstractAdmin
+{
```

Comment:
Actually, it's not the right entity. You're right and we should do this on TonMacronChoiceAdmin.php, but this entity should be ordered by createdAt DESC.